packages/mobile/src/screens/chat-screen/ReactionPopup.tsx

```diff
@@ -95,24 +99,47 @@ export const ReactionPopup = ({
     (r) => r.user_id === userIdEncoded
   )?.reaction
 
-  const beginAnimation = useCallback(() => {
-    Animated.spring(backgroundOpacityAnim.current, {
+  const beginMountAnimation = useCallback(() => {
+    Animated.timing(backgroundOpacityAnim.current, {
```

Comment:
Now that I see this, can you use the Animated.parallel composer function to just have one .start()?